.github/scripts/trigger-and-wait.sh

```diff
@@ -87,11 +87,14 @@ trigger_workflow() {
     -H "Content-Type: application/json" \
     -H "Authorization: Bearer ${INPUT_GITHUB_TOKEN}" \
     --data "{\"ref\":\"${ref}\",\"inputs\":${inputs}}"
+  
+  # Sleep after triggering workflow so it can be polled for status
+  echo "Sleeping for $wait_interval seconds"
+  sleep $wait_interval
 }
 
 wait_for_workflow_to_finish() {
   # Find the id of the last build
-  sleep "${wait_interval}" # Let workflow be triggered before fetching the latest workflow
```

Comment:
Removing extra 10seconds sleep, and keeping in sync with original author's variant.